yarn.lock

```diff
@@ -11776,6 +11777,13 @@ ovh-ui-kit@^2.34.1:
   dependencies:
     less-plugin-remcalc "^0.1.0"
 
+ovh-ui-kit@^2.34.3:
+  version "2.34.3"
```

Comment:
We should check that this new entry in the `yarn.lock` doesn't conflict with any other else and all versions are the same

Reply:
Regarding the `yarn.lock` version @ `master` branch:


```txt
...
ovh-ui-kit@^2.23.1, ovh-ui-kit@^2.33.4, ovh-ui-kit@~2.33.4:
  version "2.33.4"
  resolved "https://registry.yarnpkg.com/ovh-ui-kit/-/ovh-ui-kit-2.33.4.tgz#8b620ed078c1fb9231bafc89e5575895d7b38fbd"
  integrity sha512-eMQlEoWF3Aopp4IEQOJBKzTHvTd4t0I5BZBrtWGoJpfURQf8EI/FR4B/jry5PJf8KcAuw8EX6ICZKZsZvlQAYw==
  dependencies:
    less-plugin-remcalc "^0.1.0"

ovh-ui-kit@^2.34.1:
  version "2.34.1"
  resolved "https://registry.yarnpkg.com/ovh-ui-kit/-/ovh-ui-kit-2.34.1.tgz#759d16ee0e7b1feed720fb220ed97e8d12c2c03b"
  integrity sha512-Q1xSTN0Cld7nC9bHHJxXEUBkaC9LmKYotF8a6e4c32SWWYKFGQXPIRb554u5SPrTwxLhhESuaeVY1mT9PuDzmQ==
  dependencies:
    less-plugin-remcalc "^0.1.0"

"ovh-ui-kit@git+https://github.com/ovh-ux/ovh-ui-kit.git":
  version "0.0.0"
  resolved "git+https://github.com/ovh-ux/ovh-ui-kit.git#1b4b27fc073ab81fe1c8f24f115ebb7af31e5d6e"
...
```

Here is the diff: https://github.com/ovh-ux/ovh-ui-kit/compare/v2.34.1...v2.34.3

It brings only patches.

Nothing blocker.

Reply:
Rebased on `develop` due to #1403.